pages/resources.vue

```diff
@@ -1,33 +1,41 @@
- <template>
-    <div id="main-content" class="text-wrapper">
-        <h1 class="text-sm md:text-xl lg:text-2xl mb-3">Resources</h1>
-        <p class="mb-3"><a href="ArcGIS Online Files.zip">Download zipfile of CSV files for ArcGIS Online</a></p>
-        <p class="mb-3"><a href="https://docs.google.com/spreadsheets/d/14W8cWmqTaTgkwTFOjMYzV-BzqJgrGz8dKafmGfeuFPs/edit?usp=sharing" target="_blank">Spreadsheet of unfound place-names</a></p>
-        <p class="mb-3"><a href="https://docs.google.com/document/d/14JcTa6twIjsx69ifoZxVf-VaXR7nFmmQhu52zXUYUD4/edit?usp=sharing" target="_blank">Items added to Wikidata word document</a></p>
-        <p class="mb-3"><a href="https://w.wiki/6rX" target="_blank">Query for accused witches residence map</a></p>
-    </div>
+<template>
+  <div id="main-content" class="text-wrapper">
+    <h1 class="text-sm md:text-xl lg:text-2xl mb-3">Resources</h1>
+    <p class="mb-3">
+      <a href="ArcGIS Online Files.zip" target="_blank" download
```

Comment:
I think because we are hosting this on linux, we should change the file name to not have spaces. Something like `arcgis_online_files.zip` might suffice.

Also is the target definitely needed? I think we might be able to get this working without this required. Thanks Ruby

Reply:
done, had just seen the target blank recommended as a fix online but seems to be working fine without and have changed name

Reply:
Thanks Ruby. Looks good to me!